code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1043 +1,992 @@
-var/list/sacrificed = list()
+/var/list/sacrificed = list()
+var/list/non_revealed_runes = (subtypesof(/obj/effect/rune) - /obj/effect/rune/malformed)
 
+/*
+This file contains runes.
+Runes are used by the cult to cause many different effects and are paramount to their success.
+They are drawn with an arcane tome in blood, and are distinguishable to cultists and normal crew by examining.
+Fake runes can be drawn in crayon to fool people.
+Runes can either be invoked by one's self or with many different cultists. Each rune has a specific incantation that the cultists will say when invoking it.
+To draw a rune, use an arcane tome.
+*/
 
-/////////////////////////////////////////FIRST RUNE
-/obj/effect/rune/proc/teleportRune()
-	return teleport(src.word3)
-
-/obj/effect/rune/proc/teleport(var/key)
-	var/mob/living/user = usr
-	var/allrunesloc[]
-	allrunesloc = new/list()
-	var/index = 0
-//	var/tempnum = 0
-	for(var/obj/effect/rune/R in world)
-		if(R == src)
-			continue
-		if(R.word1 == cultwords["travel"] && R.word2 == cultwords["self"] && R.word3 == key && is_teleport_allowed(R.z))
-			index++
-			allrunesloc.len = index
-			allrunesloc[index] = R.loc
-	if(index >= 5)
-		to_chat(user, "\red You feel pain, as rune disappears in reality shift caused by too much wear of space-time fabric")
-		if(istype(user, /mob/living))
-			user.take_overall_damage(5, 0)
+/obj/effect/rune
+	name = "rune"
+	var/cultist_name = "basic rune"
+	desc = "An odd collection of symbols drawn in what seems to be blood."
+	var/cultist_desc = "a basic rune with no function." //This is shown to cultists who examine the rune in order to determine its true purpose.
+	anchored = 1
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	var/visibility = 0
+	var/view_range = 7
+	unacidable = 1
+	layer = TURF_LAYER
+
+	var/invocation = "Aiy ele-mayo!" //This is said by cultists when the rune is invoked.
+	var/req_cultists = 1 //The amount of cultists required around the rune to invoke it. If only 1, any cultist can invoke it.
+	var/rune_in_use = 0 // Used for some runes, this is for when you want a rune to not be usable when in use.
+
+	var/scribe_delay = 50 //how long the rune takes to create
+	var/scribe_damage = 0.1 //how much damage you take doing it
+
+	var/allow_excess_invokers = 0 //if we allow excess invokers when being invoked
+	var/construct_invoke = 1 //if constructs can invoke it
+
+	var/req_keyword = 0 //If the rune requires a keyword - go figure amirite
+	var/keyword //The actual keyword for the rune
+
+
+/obj/effect/rune/New(loc, set_keyword)
+	..()
+	if(set_keyword)
+		keyword = set_keyword
+	check_icon()
+	var/image/blood = image(loc = src)
+	blood.override = 1
+	for(var/mob/living/silicon/ai/AI in player_list)
+		AI.client.images += blood
```

Comment:
@Fox-McCloud eh should i remove this..from what i can gather it just makes runes look like bloods spatter to silicons/ai instead of runes...